hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/AvailabilityConditionEval.java

```diff
@@ -31,6 +31,8 @@
  */
 public class AvailabilityConditionEval extends ConditionEval {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
I'm not an expert but this is my understanding:  It's not mandatory as a value will be generated if not specified.  But the generated value could change based on class changes that may not have actually made the Object serialization fail, so an unnecessary incompatibility.  Making it explicit just puts it under our control.  It doesn't affect xml/json-style serialization, just the built in Java Object serialization, so it's sort of a no-op for us at the moment, I think.  Anyway, it makes the Eclipse warning go away :)  But did you want me to remove them?